Shared/ViewModels/AdminDashboard/ServerUsersViewModel.swift

```diff
@@ -144,6 +186,21 @@ final class ServerUsersViewModel: ViewModel, Eventful, Stateful, Identifiable {
         }
     }
 
+    // MARK: - Refresh User
+
+    private func refreshUser(_ userID: String) async throws {
+        let request = Paths.getUserByID(userID: userID)
+        let response = try await userSession.client.send(request)
+
+        let newUser = response.value
+
+        await MainActor.run {
+            if let index = self.users.firstIndex(where: { $0.id == userID }) {
```

Comment:
Looks like we should migrate `users: [UserDto]` to `users: IdentifiedArrayOf<UserDto>`. I believe the impact should be minimal.